src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/televisions/{model}")
+    @ResponseBody
+    public String getTelevision(@PathVariable String model) {
+        return "Type television" + model;
+    }
+
+    @PostMapping("/api/television")
```

Comment:
Je GetMappings beginnen met "/television", waarom begint deze met "/api"? Het is belangrijk om consequent en restful te zijn in je paden